src/app/components/radio/index.html

```diff
@@ -1,13 +1,269 @@
-  </sky-demo-page-summary>
+The radio button component is a selection input for choosing one option from a small set.
+</sky-demo-page-summary>
+    
+<div class="sky-margin-stacked-content-section">
+    <sky-demo-page-example>
+        <sky-radio-demo></sky-radio-demo>
+    </sky-demo-page-example>
+</div>
 
+<sky-tabset id="content-tabs">
+	
+	<sky-tab tabHeading="Design">
+        
+        <sky-demo-page-content sectionHeading="Usage">
+            
+            <h3 class="sky-text-success">Use when</h3>
+            
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                    <p>Use radio buttons when the user must choose one option from a set of 2-5. Use radio buttons in a modal, where the user will confirm the decision before changes take effect.</p>
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div>
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-1.png" />
+                    <div class="caption caption-success sky-text-success">Do use radio buttons for single selections when the number of choices is low.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+            
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                    <p>Use an icon radio group when choices can be clearly communicated with icons, and can be combined under a single field or question. </p>
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div>
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-3.png" />
+                    <div class="caption caption-success sky-text-success">Do use icon radio groups when choices can be clearly illustrated with icons.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+            
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                    <p>Use individual icon radio buttons when the radio group is spread across multiple rows or objects.</p>
```

Comment:
Just double-checking that we want to keep this guideline as a recommended use case or icon radio buttons. I thought we discussed this offline and were considering removing this guideline.

Reply:
I think I dropped this example in the last update, or at least I tried. We are NOT going to advise using unconnected radio buttons in this way. When we get to Phase 2 for radio buttons, I'm going to recommend dropping the unconnected radio buttons from the demo, as well.

Reply:
Looks like I had my wires crossed last night and that you did remove this example. I must have gotten mixed up going back and forth between the PR, the PR build, the live docs, and a locally served instance because I don't see this guideline in place anymore. Sorry about that.